labeler/has_pull_request.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/parkr/auto-reply/ctx"
 )
 
-var fixesIssueMatcher = regexp.MustCompile(`Fixes #(\d+)`)
+var fixesIssueMatcher = regexp.MustCompile(`(Clos|Fix|Resolv)(e|es|ed) #(\d+)`)
```

Comment:
This is good, but being explicit might make the code easier to read. For example, this will never match `Fix` because it expects `Fixe` :wink:

Also, we could make this a non-capturing group. Not sure how big of a deal that is performance wise in Go.

```go
var fixesIssueMatcher = regexp.MustCompile(`(?:Close|Closes|Closed|Fix|Fixes|Fixed|Resolve|Resolves|Resolved) #(\d+)`)
```